CHANGELOG.md

```diff
@@ -1,5 +1,64 @@
+- Added bottom tab bar for Brave Wallet. ([#8166](https://github.com/brave/brave-ios/issues/8166))
+- Added the ability to group NFTs in Brave Wallet. ([#8073](https://github.com/brave/brave-ios/issues/8073))
+- Added new design for the Portfolio page. ([#8072](https://github.com/brave/brave-ios/issues/8072))
+- Added method to return "Swap" protocol fees. ([#7924](https://github.com/brave/brave-ios/issues/7924))
+- Added support for Sign-In with Ethereum (EIP-4361). ([#7827](https://github.com/brave/brave-ios/issues/7827))
+- Added support for EIP-6963. ([#7779](https://github.com/brave/brave-ios/issues/7779))
+- Added the ability for users to navigate to ENS domains. ([#6225](https://github.com/brave/brave-ios/issues/6225))
+- Added the ability to select a specific RPC URL as active for Brave Wallet. ([#5902](https://github.com/brave/brave-ios/issues/5902))
+- [Security] Updated Brave Wallet password policy. ([#6093](https://github.com/brave/brave-ios/issues/6093))
+- Implemented "Bridge to Aurora" for Brave Wallet. ([#5785](https://github.com/brave/brave-ios/issues/5785))
+- Updated Portfolio settings page in Brave Wallet. ([#8089](https://github.com/brave/brave-ios/issues/8089))
+- Updated button to create wallet to be disabled when wallet is already being created. ([#8357](https://github.com/brave/brave-ios/issues/8357))
+- Updated “Send” screen in Brave Wallet to not show decimal values for zero balances. ([#8232](https://github.com/brave/brave-ios/issues/8232))
+- Updated “Save Changes” button to be enabled only when there is a change in wallet portfolio settings. ([#8229](https://github.com/brave/brave-ios/issues/8229))
+- Fixed “view my recovery phrase” not displaying in Brave Wallet. ([#8445](https://github.com/brave/brave-ios/issues/8445))
+
+### General 
+
+- Added support for all history data to be synced when "History" sync is enabled. ([#8305](https://github.com/brave/brave-ios/issues/8305))
+- Added setting to ensure YouTube links always load in brave. ([#8272](https://github.com/brave/brave-ios/issues/8272))
+- Added confirmation message for successfully exporting bookmarks. ([#8236](https://github.com/brave/brave-ios/issues/8236))
+- Added confirmation message for resetting configuration in VPN settings. ([#8087](https://github.com/brave/brave-ios/issues/8087))
+- Added support for additional schemas for going to the App Store. ([#8035](https://github.com/brave/brave-ios/issues/8035))
+- Added VPN improvements for callouts and the settings page. ([#4879](https://github.com/brave/brave-ios/issues/4879))
+- Added the ability to download .mobileconfig files and auto-import into settings app. ([#4358](https://github.com/brave/brave-ios/issues/4358))
+- Implemented tabs bar re-order via drag and drop. ([#8058](https://github.com/brave/brave-ios/issues/8058))
+- Removed SDWebImage SVG coder. ([#7026](https://github.com/brave/brave-ios/issues/7026))
```

Comment:
Lets mention why it was removed. Also mention it was replaced. Maybe something along the lines of the following:

```
- Replaced "SDWebImage SVG" with "Brave-Core SVG" due to "SDWebImage" crashing often when parsing SVGs.
```